### PR TITLE
Updated and added 'finalised' architecture diagram to assets folder: Andrew

### PR DESCRIPTION
**Pull request by MurasatoNaoya**
This is the architecture diagram for StarWatch project, displaying the high-level architecture of our workflow. 

The architecture diagram presents 5 of the main features / pieces of functionality of the project - 

1. A RDS Postgres database that interacts with multiple Lambda functions, external APIs and the dashboard - many of which read and write to it across the workflow. Stored data including astronomy / celestial body data, weather data, user subscription information and relevant images (NASA image of the day, etc..)

2. Two ETL pipelines triggered weekly. The first fetching and loading celestial body positional data for all available bodies from the astronomyAPI, and the second  for both sunrise and sunset times; both for the same one week period and run in parallel. Using data from both pipelines, visible celestial bodies at night (after sunset) can be identified. 

3.  An ETL pipeline that triggers quadhourally, fetching weather data for all specified counties and loading it into the dedicated RDS. Allowing amateur astronomers to account for environmental factors that may affect astronomical visibility. 

4. A dashboard for visualising all the data and images sourced from the many APIs used for this project. Presenting them in an appealing and easy to understand manner. Amateur astronomers are also able to use their email and/or phone number to subscribe to an SNS service in the dashboard, so they can be notified about visible celestial objects and/or events near them.

5. Two ETL pipelines triggering hourly. The first fetches UK aurora data from the AuroraWatch API, while the second runs a script for finding all visible celestial bodies and events relative to subscribers' counties. These hourly pipelines also run in parallel for efficiency purposes.

This pull request addresses and closes the following issues: #68 #80

### CODE REVIEW

File: assets/starwatch_architecture_diagram.png
Comment (CalebMathers) [inline assets/starwatch_architecture_diagram.png:1]: Good size now, thank you, very clear!

**Comment by olliejth** [comment]
We'll need to make sure this stays up to date if the proposed architecture changes during the project duration.

**Review by CalebMathers — APPROVED** [review]
Good architecture diagram

**Review by AlexOBarnes — APPROVED** [review]
Much clearer image thank you

**Pull request merged**